Makefile

```diff
@@ -3,16 +3,26 @@ export GOPATH=/tmp/go
 endif
 
 INSTALL := /usr/bin/install
+DBDIR := /var/run/redis/sonic-db/
 
 all: sonic-telemetry
 
 sonic-telemetry:
+# copy sonic-telemetry source code into ${GOPATH}/src directory for building, otherwise it is not using committed codes
+	mkdir -p ${GOPATH}/src/github.com/Azure
+	rsync -a ../sonic-telemetry ${GOPATH}/src/github.com/Azure/
```

Comment:
>rsync [](start = 1, length = 5)

Why not use `cp` ?

Reply:
personally I like to use rsync, cp is also fine. Updated